sections/Design.tex

```diff
@@ -5,37 +5,83 @@ \section{Philosophy of the language}
 
 \section{Generel}
 \subsection{Character set}
+Source code must be in Unicode.
+This is chosen even though all variables and types is in ACSI, because Unicode gives the programmer of different origins the possibility to use special characters of theirs first language.
+An example of this would be the danish %\"\Æ,\Ø,\Å\".
 
 \subsection{Comments}
+Comments are done as in other large languages i.e. C and Java, with the use of \"\/\/\".
+The compiler ignores everything from \"\/\/\" to the end of the line.
+The decision  to use already existing solutions is to make sure the language is easy to learn if the programmer already have some coding experience.
+Below on \myref{lst:CommSemi} a snippet of an comment in use can be seen.
+
+\subsection{Semicolon}
+A semicolon \"\;\" is used as a terminator and are required after every statement.
+This is again based on other already existing languages such as Java and the C family.
+The use of terminators in a language is of less importance, and therefore we decide that a semicolon is required to terminate each statement for easier readability.
+The use of semicolon can also be found on \myref{lst:CommSemi}.
+
+
+\begin{lstlisting}[caption={Comment and Semicolon},label={lst:CommSemi}]
+int a = 1;		//initialization with specified type.
+\end{lstlisting}
 
 \subsection{Case Sensitivity}
+The language is case sensitive to make sure the programmer uses whatever convention he is most use to, and pleased with.
+C\# uses case sensitivity to i.e. allow the programmer to separate class names and instances.
+Having a case sensitive language provide a higher readability and writability since the programmer are use to writing English or other spoken language which are case sensitivity.

```

Comment:
Lyder lidt underligt, ændre istedet slutningen til:
[...] since it relates to how common languanges such as english are written.